addons/point_of_sale/migrations/13.0.1.0.1/end-migration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 ForgeFlow <https://www.forgeflow.com>
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def create_pos_payment_methods(env):
+    env["pos.config"].post_install_pos_localisation()
+
+
+def create_pos_payments(env):
+    env.cr.execute("""
+        SELECT DISTINCT absl.id, ppm.id
+        FROM account_bank_statement abs
+        JOIN pos_session ps ON abs.pos_session_id = ps.id
+        JOIN account_bank_statement_line absl ON absl.statement_id = abs.id
+        LEFT JOIN account_journal aj ON abs.journal_id = aj.id
+        JOIN pos_payment_method ppm ON ppm.cash_journal_id = aj.id
+    """)
```

Comment:
I think you need to add
` WHERE  absl.pos_statement_id is not NULL`

Also, in order to avoid the browse, you can return also `absl.create_date`, `absl.amount` and `absl.name` WDYT?

Reply:
done